src/components/DocumentationTopic.vue

```diff
@@ -539,9 +539,12 @@ export default {
     declarations({ primaryContentSections = [] }) {
       return primaryContentSections.filter(({ kind }) => kind === SectionKind.declarations);
     },
-    hasOtherDeclarations({ declarations = [] }) {
-      // there's always only 1 `declaration` at this level
-      return declarations.length ? declarations[0].declarations.some(decl => Object.prototype.hasOwnProperty.call(decl, 'otherDeclarations')) : false;
```

Comment:
Could we eliminate the ternary expression and make this simpler so that it's just

```
declarations.length && !enableMinimized && declarations[0].declarations.some(...)
```

?

Reply:
Good point!